src/components/pages/Speakers.js

```diff
@@ -17,9 +17,7 @@ function Events() {
 
     return (
         <div className="container py-5">
-            <div className="speakers-title header-gradient pt-5">
-                <h1 className="">Speakers</h1>
-            </div>
+            <h1 className="">Speakers</h1>
```

Comment:
I think we can comment out this heading since the nav bar is underlined above too? But if you want to keep it that works too